tutorials/bios-boot-tutorial/bios-boot-tutorial.py

```diff
@@ -351,6 +351,11 @@ def stepSetSystemContract():
     retry(args.cleos + 'push action eosio activate \'["6bcb40a24e49c26d0a60513b6aeb8551d264e4717f306b81a37a5afb3b47cedc"]\' -p eosio@active')
     # BLS_PRIMITIVES
     retry(args.cleos + 'push action eosio activate \'["98c4175db53ed27e7911a1b5adf0e7db0fc96c2cae172cf594dfa9a742ca9e88"]\' -p eosio@active')
+    # DISALLOW NEW DEFERRED TRANSACTIONS
+    retry(args.cleos + 'push action eosio activate \'["440c3efaaab212c387ce967c574dc813851cf8332d041beb418dfaf55facd5a9"]\' -p eosio@active')
```

Comment:
I believe this is the `description digest` not the `feature digest`.  I think we're looking for: `fce57d2331667353a0eac6b4209b67b843a7262a848af0a49a6e2fa9f6584eb4`

Same as below